### ISSUE TITLE
Some questions about the size and partitioning of data sets

### ISSUE THREAD

**Issue by parZival27** [google-research-datasets/ToTTo#2]
Hello, I have tried to count the total number of Headers in the training set. After adjusting the case, the number is 145570, and the number given in the paper is 120761. Besides, in the validation set, 3916 cases of data are labeled as non-overlap, but when I judge the relationship between the header set of the training set and the non-overlap data set, only 1938 cases of data meets the standard of non-overlap describing in the paper. May I ask whether I have omitted some pretreatment methods? What is the practical implementation method for dividing overlap/non-overlap? Thank you very much!
